db/migrate/20201030020815_add_lock_unlock_to_post.rb

```diff
@@ -0,0 +1,12 @@
+class AddLockUnlockToPost < ActiveRecord::Migration[5.1]
+  def change
+    add_column :posts, :locked_by, :integer
+    add_column :posts, :locked_at, :datetime
+    add_column :posts, :lock_reason, :integer
+    add_index :posts, :locked_by
+
+    add_column :posts, :unlocked_by, :integer
+    add_column :posts, :unlocked_at, :datetime
+    add_index :posts, :unlocked_by
```

Comment:
This seems like it confuses matters. I think we should just unset the locked columns when we unlock, and worry about a proper audit log later

Reply:
Does rails admin only produce edit histories if it was edited within rails admin?

Reply:
Ok, wasn't sure how we wanted to track this really.